lib/options.json

```diff
@@ -390,6 +393,7 @@
       "ca": "should be {String|Buffer}",
       "cert": "should be {String|Buffer}",
       "clientLogLevel": "should be {String} and equal to one of the allowed values\n\n [ 'info', 'warn', 'error', 'debug', 'trace', 'silent' ]\n\n (https://webpack.js.org/configuration/dev-server/#devserverclientloglevel)",
+      "clientMode": "should be {String} (https://webpack.js.org/configuration/dev-server/#devserverclientmode-)",
```

Comment:
extra `-` at end of line

Reply:
Do we need to modify this? L437 was fixed it.

Reply:
> Do we need to modify this? L437 was fixed it.

Yes, I mistakenly did not remove that `-`, but they should be removed in both places.